core/Symbols.h

```diff
@@ -59,14 +59,21 @@ class Method final {
         bool isFinal : 1;
         bool isOverride : 1;
         bool isIncompatibleOverride : 1;
+
+        constexpr static uint16_t NUMBER_OF_FLAGS = 10;
+        constexpr static uint16_t VALID_BITS_MASK = (1 << NUMBER_OF_FLAGS) - 1;
         Flags() noexcept
             : isRewriterSynthesized(false), isProtected(false), isPrivate(false), isOverloaded(false),
               isAbstract(false), isGenericMethod(false), isOverridable(false), isFinal(false), isOverride(false),
               isIncompatibleOverride(false) {}
 
         uint16_t serialize() const {
```

Comment:
Note to self: _before_ merging, write unit tests that check the serialize methods.